lib/Version.cjs

```diff
@@ -1,71 +1,85 @@
-var { Platforms } = require("../dist/index.cjs");
-var _ = require("lodash");
-var semver = require("semver");
+/**
+ * @file    [Description of file purpose]
+ * @author  ayushmxn
+ */
+
+const _ = require('lodash');
+const semver = require('semver');
+
+const { Platforms } = require('../dist/index.cjs');
 
 /**
  * Represents a version.
- * @constructor
- * @param {string} args.version.
- * @param {string} args.flavors.
- * @param {array} args.supportedPlatforms - array of strings like `osx64`.
- * @param {string} args.downloadUrl.
+ *
+ * @param  {object}   args                     Function options
+ * @param  {string}   args.downloadUrl         Base URL path for dowloading NW.js, such as 'https://dl.nwjs.io/'
+ * @param  {string}   args.version             An NW.js version, such as '0.12.3'
+ * @param  {string[]} args.flavors             Array of strings for NW.js flavors, such as ['normal', 'sdk']
+ * @param  {string[]} args.supportedPlatforms  Array of strings for supported platforms, such as ['osx64', 'win32']
+ * @return {object}                            Object containing platforms key with generated URLs
  */
 module.exports = function Version(args) {
-  var generatePlatformUrls,
-    result = {
-      isLegacy: semver.lt(args.version, "0.12.3"),
-      name:
-        semver.gte(args.version, "0.12.0") ||
-        semver.satisfies(args.version, "~0.12.0-alpha")
-          ? "nwjs"
-          : "node-webkit",
-      version: args.version,
-    };
+  let generatePlatformUrls;
+  let name = 'node-webkit';
+  if (
+    semver.gte(args.version, '0.12.0') ||
+    semver.satisfies(args.version, '~0.12.0-alpha')
+  ) {
+    name = 'nwjs';
+  }
+  const result = {
+    isLegacy: semver.lte(args.version, '0.12.3'),
+    name,
+    version: args.version,
+  };
 
   // 0.12.3 is an exception that is in the manifest but is pretty much a legacy version
-  if (result.isLegacy || args.version === "0.12.3") {
```

Comment:
Shot in the dark but looking at the error logs, maybe this is what is causing test 109 to fail?

Reply:
Yup, have a branch locally. Busy day will fix later